imagetest/test_suites/image_boot/image_boot_test.go

```diff
@@ -123,5 +197,49 @@ func TestBootTime(t *testing.T) {
 	if err != nil {
 		t.Fatalf("failed to convet start time %s", metadata)
 	}
-	t.Logf("image boot time is %d", time.Now().Second()-startTime)
+	t.Logf("image start time is %d", time.Now().Second()-startTime)
+}
+
+func TestBootTime(t *testing.T) {
+	image, err := utils.GetMetadata("image")
```

Comment:
What's is this utils.GetMetadata("image") returning? the image name? can you make it return os-family? if so getThresholdValue() can be simplified or wouldn't be required at all.

Reply:
no, it cannot return the family. this is the standard paradigm for this kind of check, see other tests